src/master.rs

```diff
@@ -336,6 +373,15 @@ impl Master {
     // reference_clock_time, sync_monitor_queue, sync_monitor_process
 }
 
+fn raw_array_to_string(arr: &[i8; 64]) -> String {
+    let s = arr
+        .iter()
+        .filter(|x| **x != 0)
```

Comment:
why are NULs filtered, is it not a NUL terminated string?

Reply:
You're probably right ;-) I'll have second look at it.

Reply:
No I use your implementation and it seems to work :)
```rust
unsafe { CStr::from_ptr(data).to_string_lossy().into_owned() }
```